ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/ip-dhcp-snooping.cfg

```
@@ -0,0 +1,19 @@
+!RANCID-CONTENT-TYPE: arista
+!
+ip dhcp snooping bridging
+ip dhcp snooping information option circuit-id type 10 format %h:%p
+ip dhcp snooping vlan 10,20,500,1000-2000
```

I think this should render 

```suggestion
ip dhcp snooping bridging
ip dhcp snooping information option
ip dhcp snooping information option circuit-id type 10 format %h:%p
ip dhcp snooping vlan 10,20,500,1000-2000
```

based on CLI testing (cf comment below) and this TOI: https://www.arista.com/en/support/toi/eos-4-25-2f/14713-dhcp-snooping

---

@gmuloc you are absolutely correct. I justed checked it on a switch right now too.
I am sorry, I missed that in my tests.

I revised it:
```
{#
    Copyright (c) 2023 Arista Networks, Inc.
    Use of this source code is governed by the Apache License 2.0
    that can be found in the LICENSE file.
#}
{# eos - ip dhcp snooping #}
{% if ip_dhcp_snooping is arista.avd.defined %}
{%     if ip_dhcp_snooping.enabled is arista.avd.defined(true) %}
{%         if ip_dhcp_snooping.bridging is arista.avd.defined(true) %}
!
ip dhcp snooping bridging
{%         else %}
!
ip dhcp snooping
{%         endif %}
{%         if ip_dhcp_snooping.information_option.enabled is arista.avd.defined(true) %}
ip dhcp snooping information option
{%             if ip_dhcp_snooping.information_option.circuit_id_type is arista.avd.defined and ip_dhcp_snooping.information_option.circuit_id_type_format is arista.avd.defined %}
ip dhcp snooping information option circuit-id type {{ ip_dhcp_snooping.information_option.circuit_id_type }} format {{ ip_dhcp_snooping.information_option.circuit_id_type_format }}
{%             endif %}
{%         endif %}
{%         if ip_dhcp_snooping.vlan is arista.avd.defined %}
ip dhcp snooping vlan {{ ip_dhcp_snooping.vlan }}
{%         endif %}
{%     endif %}
{% endif %}
```

```
ip_dhcp_snooping:
  enabled: true
  bridging: true
  information_option:
    enabled: true
    circuit_id_type: 10
    circuit_id_type_format: '%h:%p'
  vlan: "10,20,500,1000-2000
  ```
  
  ```
ip dhcp snooping bridging
ip dhcp snooping information option
ip dhcp snooping information option circuit-id type 10 format %h:%p
ip dhcp snooping vlan 10,20,500,1000-2000
```

test on 720XP:
```
ccess-10-014-01__12:03:09(config)#
access-10-014-01__12:03:09(config)#ip dhcp snooping bridging
access-10-014-01__12:03:53(config)#ip dhcp snooping information option
access-10-014-01__12:03:53(config)#ip dhcp snooping information option circuit-id type 10 format %h:%p
access-10-014-01__12:03:53(config)#ip dhcp snooping vlan 10,20,500,1000-2000
access-10-014-01__12:03:54(config)#
access-10-014-01__12:03:55#show ip dhcp snooping
DHCP Snooping is enabled
DHCP Snooping is operational
DHCP Snooping is configured on following VLANs:
  10,20,500,1000-2000
DHCP Snooping bridging is operational on following VLANs:
  10,20,500,1000-2000
Insertion of Option-82 is enabled
  Circuit-id default sub-option Type: 10
  Circuit-id default format: Hostname:Interface name
  Remote-id: ac:3d:94:c8:0b:10 (Switch MAC)
access-10-014-01__12:04:01#
```

---

fixed